ui/src/onboarding/containers/LoginPage.scss

```diff
@@ -0,0 +1,166 @@
+.clockface--app-wrapper.sign-up--page {
+  background-image: linear-gradient(
+    47deg,
+    rgba(19, 0, 45, 0) 65%,
+    rgba(191, 47, 229, 0.4) 100%
+  ) !important;
+  background-color: #13002d !important;
+}
```

Comment:
might be worth doing `div.clockface--app-wrapper`, may not be possible if more than divs are used